package.json

```diff
@@ -76,11 +72,10 @@
     "conventional-changelog-cli": "1.2.0",
     "conventional-changelog-lint": "1.0.0",
     "dive": "0.4.0",
+    "eslint": "2.13.1",
```

Comment:
Oh, and I forgot that I should throw a comment on here...

I'm not sure what version of ESLint to use. Per http://eslint.org/docs/user-guide/migrating-to-3.0.0

> With ESLint v3.0.0, we are dropping support for Node.js versions prior to 4. Node.js 0.10 and 0.12 are in maintenance mode and Node.js 4 is the current LTS version. If you are using an older version of Node.js, we recommend upgrading to at least Node.js 4 as soon as possible. If you are unable to upgrade to Node.js 4 or higher, then we recommend continuing to use ESLint v2.x until you are ready to upgrade Node.js.

> Important: We will not be updating the ESLint v2.x versions going forward. All bug fixes and enhancements will land in ESLint v3.x.

I wasn't 100% on our support for Node < 4 (the [package.json](https://github.com/mozilla-jetpack/jpm/blob/413022d1c647565d24090061c1c5898ae092d24d/package.json#L9-L12) mentions `node>=0.10`), so I left this as **eslint@2**. If we're only supporting Node >= 4, we should probably upgrade to **eslint@3**.